src/main_app/db/services/settings_service.py

```
@@ -98,16 +98,20 @@ def update_setting(
         }
         if title:
             data["title"] = title
-
-        return self.update(record, **data)
+        try:
+            self.update(record, **data)
+            return True
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)